scripts/upgrade

```diff
@@ -90,7 +90,7 @@ then
 	ynh_script_progression --message="Upgrading source files..." --weight=1
 
 	# Download, check integrity, uncompress and patch the source from app.src
-	ynh_setup_source --dest_dir="$final_path" --keep="config/config.js"
+	ynh_setup_source --dest_dir="$final_path" --keep="config/config.js /customize"
```

Comment:
I don't know if we need `/`

Reply:
I am talking about a folder needed to customize the application, in my instance I have edited few files in this directory... https://docs.cryptpad.org/fr/admin_guide/customization.html#look-and-feel